### PR TITLE
update picard/collectwgsmetrics

### PR DESCRIPTION
**Pull request by matthdsm**
Add optional bam index
Add meta to fasta

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by nvnieuwk — APPROVED** [review]
LGTM

File: modules/nf-core/picard/collectwgsmetrics/meta.yml [hunk 1]
```diff
@@ -22,8 +22,16 @@ input:
         e.g. [ id:'test', single_end:false ]
   - bam:
       type: file
-      description: BAM file
-      pattern: "*.{bam}"
+      description: Aligned reads file
+      pattern: "*.{bam, cram}"
```
Comment (maxulysse) [inline modules/nf-core/picard/collectwgsmetrics/meta.yml:26]: no sam here?

Comment (maxulysse) [inline modules/nf-core/picard/collectwgsmetrics/meta.yml:26]:
```suggestion
      pattern: "*.{bam,cram}"
```

File: modules/nf-core/picard/collectwgsmetrics/meta.yml [hunk 2]
```diff
@@ -22,8 +22,16 @@ input:
...
+  - bai:
+      type: file
+      description: (Optional) Aligned reads file index
+      pattern: "*.{bai, crai}"
```
Comment (maxulysse) [inline modules/nf-core/picard/collectwgsmetrics/meta.yml:30]:
```suggestion
      pattern: "*.{bai,crai}"
```

**Comment by maxulysse** [comment]
Why no tests?

**Comment by matthdsm** [comment]
Closing this and merged with collectHSmetrics because the subwf depends on both

**Pull request closed without merging**